pkg/executables/cmk.go

```diff
@@ -144,11 +144,17 @@ func (c *Cmk) ValidateDiskOfferingPresent(ctx context.Context, zoneId string, di
 	}
 	offerings := response.CmkDiskOfferings
 	if len(offerings) > 1 {
-		return fmt.Errorf("duplicate disk offering %s found", diskOffering)
+		return fmt.Errorf("duplicate disk offering ID/Name %s/%s found", diskOffering.Id, diskOffering.Name)
 	} else if len(offerings) == 0 {
-		return fmt.Errorf("disk offering %s not found", diskOffering)
+		return fmt.Errorf("disk offering ID/Name %s/%s not found", diskOffering.Id, diskOffering.Name)
 	}
 
+	if offerings[0].Customized && diskOffering.Size <= 0 {
+		return fmt.Errorf("disk offering size %d <= 0 for customized disk offering", diskOffering.Size)
+	}
```

Comment:
isn't there also a limit on the capc diskoffering as well? LIke an upper/lower limit on the diskoffering when it's requested? I think it'd be good to check to make sure we're within those bounds as well

Reply:
I did not find lower/upper limit on the custom diskoffering.
here is the api call return for diskoffering: 
```
{
      "created": "2022-01-13T20:01:51-0500",
      "disksize": 0,
      "displayoffering": true,
      "displaytext": "Custom Disk",
      "id": "df36c956-9f4a-4cc3-84c9-239e0914d0f8",
      "iscustomized": true,
      "name": "Custom",
      "provisioningtype": "thin",
      "storagetype": "shared"
    }```